docs/content/any/project/changelogs/NEXT.md

```diff
@@ -21,9 +21,11 @@ draft: true
   before upgrading.
 {{% /callout %}}
 
-##### Breaking change 1
+##### Updated Go type checking behavior
 
-Summary of breaking change.
+When evaluating whether a given query variable matches a Go type Polar will now use direct instance comparisons instead of Go's `reflect.IsConvertableTo` functionality. This change resolves false-positive type checking results where discrete structs with identical sets of fields were considered to be equivalent. 
```

Comment:
```suggestion
When evaluating whether a given query variable matches a Go type Polar will now use direct instance comparisons instead of Go's `reflect.ConvertibleTo` functionality. This change resolves false-positive type checking results where discrete structs with identical sets of fields were considered to be equivalent. 
```